core/src/test/java/nl/nn/adapterframework/http/HttpSenderTest.java

```diff
@@ -1020,7 +1060,6 @@ public void specialCharactersDoubleEscaped() throws Throwable {
 		assertEqualsIgnoreCRLF(getFile("specialCharactersDoubleEscaped.txt"), result.trim());
 	}
 
-	@Test(expected = SenderException.class)
 	public void unsupportedScheme() throws Throwable {
```

Comment:
Moet hier nu niet weer een `@Test` annotation bij?

Reply:
Ja potverdikkie! waarom is die nou weer weg